jenkins/ci.suse.de/pipelines/openstack-crowbar.Jenkinsfile

```diff
@@ -154,9 +154,7 @@ pipeline {
            // This step does the following on the admin node:
            //  - sets up SLES and Cloud repositories
            //  - installs crowbar
-           ardana_lib.ansible_playbook('deploy-crowbar', '-e \'qa_crowbarsetup_cmd="onadmin_runlist prepareinstallcrowbar"\'')
-           ardana_lib.ansible_playbook('deploy-crowbar', '-e \'qa_crowbarsetup_cmd="onadmin_runlist bootstrapcrowbar"\'')
-           ardana_lib.ansible_playbook('deploy-crowbar', '-e \'qa_crowbarsetup_cmd="onadmin_runlist installcrowbar"\'')
+           ardana_lib.ansible_playbook('install-crowbar')
```

Comment:
Can we rename this lib? It's a bit confusing to see a ardana lib in crowbar code.

Reply:
Yeah, I know, it's really annoying having Crowbar code labeled Ardana, and it's more extensive than you can imagine. It's not really part of this PR, but we do have a Jira card tracking this type of refactoring work (https://jira.suse.de/browse/SCRD-8475 ).